pastepwn/analyzers/emailpasswordpairanalyzer.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+from .regexanalyzer import RegexAnalyzer
+
+_EMAIL_PASSWORD_REGEX = r'[\w\.\+_-]+@[\w\._-]+\.[a-zA-Z]*\:[\w\.\+\!\$\#\^&\*\(\)\{\}\[\]\_\-\@\%\=]+$'
```

Comment:
Maybe also remove the `$` from the end, because we want to match this kind of combination anywhere in the line, or what would you say?

Reply:
I intentionally added that because otherwise it will generate false alerts like for example json files. Most password leaks are in this format only.

Reply:
I will test this with pastebin data for a few days and check how it works out. Thank you for the insight.